src/three/PNTSLoader.js

```diff
@@ -43,7 +43,26 @@ export class PNTSLoader extends PNTSLoaderBase {
 				} );
 
 				const geometry = new BufferGeometry();
-				geometry.setAttribute( 'position', new BufferAttribute( POSITION, 3, false ) );
+				if ( POSITION ) {
+
+  				geometry.setAttribute( 'position', new BufferAttribute( POSITION, 3, false ) );
+
+				} else if ( POSITION_QUANTIZED ) {
+
+					const decodedPositions = new Float32Array( POSITION_QUANTIZED.length );
+					for ( let i = 0; i < POSITION_QUANTIZED.length / 3; i ++ ) {
```

Comment:
For the sake of performance can we premptively declare the length to iterate over:

```js
for ( let i = 0, l = POSITION_QUANTIZED.length / 3; i < l; i ++ ) ...
```